examples/panic_compose.rs

```diff
@@ -0,0 +1,50 @@
+use color_eyre::eyre::Report;
```

Comment:
Might be good to have a description for the purpose of this example? I'm guessing it's piping panic messages through to `tracing`?

Reply:
yes, exactly